plugins/star-rating/api/api.js

```diff
@@ -260,6 +261,38 @@ const widgetPropertyPreprocessors = {
             }
         });
     };
+    var nonChecksumHandler = function(ob) {
+        var events = JSON.parse(ob.params.qstring.events);
```

Comment:
It would crash process if I
* don't provide events
* provide incorrect JSON string
* provide empty events